apps/pie-docs/src/assets/styles/base/_shared.scss

```diff
@@ -0,0 +1,110 @@
+@use '../fozzie-config' as f;
+@use '../variables' as v;
+
+.c-content {
+    flex: 1;
+    background-color: f.$color-background-default;
+    border-top: 1px solid f.$color-border-default;
+
+    & a {
+        color: f.$color-content-link-distinct;
+
+        &:visited {
+            color: f.$color-content-link-visited;
+        }
+
+        &:hover,
+        &:active,
+        &:focus {
+            text-decoration: none;
+        }
+    }
+
+    & p:has(+ .c-tokensTable) {
```

Comment:
Unfortunately `:has` is not supported by enough browsers yet so we can't use it https://caniuse.com/css-has

Reply:
It might be worth us making a different ticket to revert the `:has` - it's in a few different places so might take more time looking into. What do you think?

Reply:
I'm going to tackle this in a separate PR 👍